utils/lxc/Makefile

```
@@ -103,6 +103,11 @@ define Package/lxc-unprivileged/install
 	$(INSTALL_DATA) ./files/lxc-unprivileged.defaults $(1)/etc/uci-defaults/lxc-unprivileged
 endef
 
+define Package/lxc-unprivileged/conffiles
+  /etc/subgid
+  /etc/subuid
```

```suggestion
/etc/subgid
/etc/subuid
```

spaces should be omitted for conffiles.